internal/mesh/internal/controllers/apigateways/controller.go

```diff
@@ -60,6 +61,11 @@ func (r *reconciler) Reconcile(ctx context.Context, rt controller.Runtime, req c
 		})
 	}
 
+	ports = append(ports, &pbcatalog.ServicePort{
+		Protocol:   pbcatalog.Protocol_PROTOCOL_MESH,
+		TargetPort: LANPortName,
+	})
```

Comment:
What's this being used for? I wouldn't expect the mesh port to be meaningful for an API gateway since nothing inside the mesh should be contacting the API gateway.

Acknowledging that a `mesh` port will be created implicitly for the API gateway anyways because of re-used code, but that should go away in the medium term.

Reply:
good catch, this was added when I was testing some things out and I didn't try removing it after I got things working